### PR TITLE
Page Descriptions: The PR

### PR DESCRIPTION
**Pull request by Symbiot01**

### CODE REVIEW

**Comment by Goose-Of-War** [comment]
_Man lazy with the PR title and desc 🥲_

File: templates/404.njk [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 {% set thispage = '404' %}
 {% set pagetitle = pagetitle if pagetitle else '404' %}
+{% set pagedesc = 'Errr' %}
```
Comment (Goose-Of-War) [inline templates/404.njk:5]: Errr? Not Error? ;-;

File: templates/_event.njk [hunk 2]
```diff
@@ -1,6 +1,7 @@
 {% extends '_base.njk' %}
 
 {% set thispage = 'events' %}
+{% set pagedesc = 'At MASK, we engage in a variety of activities that celebrate the creativity and beauty of manga and anime. Veiw every past events organised by us. ' %}
```
Comment (Goose-Of-War) [inline templates/_event.njk:4]: Spellings 💀

File: templates/404.njk (hunk 1 above)
Comment (Symbiot01) [reply templates/404.njk:5]: Whom are we sharing this link

File: templates/_event.njk (hunk 2 above)
Comment (Symbiot01) [reply templates/_event.njk:4]: ok

File: templates/404.njk (hunk 1 above)
Comment (Goose-Of-War) [reply templates/404.njk:5]: IDK fam. In case you misspell the link or something, you want them to see Errr instead of error? ;-;

**Review by Goose-Of-War — APPROVED** [review]
Now it feels ready

**Pull request merged**